snapcraft/cli/echo.py

```diff
@@ -70,6 +71,21 @@ def error(msg: str) -> None:
     click.echo("\033[0;31m{}\033[0m".format(msg), err=True)
 
 
+def echo_with_pager_if_needed(msg: str) -> None:
+    """Output msg to stdout using pager if output is too large for terminal."""
+    term_size = shutil.get_terminal_size()
```

Comment:
can you look at the click alternatives for this one?

Reply:
Deprecated since version 8.0: Will be removed in Click 8.1. Use shutil.get_terminal_size() instead.

Reply:
https://click.palletsprojects.com/en/8.0.x/api/#click.get_terminal_size